src/fswAlgorithms/orbitControl/lambertPlanner/lambertPlanner.h

```diff
@@ -54,6 +54,8 @@ class LambertPlanner: public SysModel {
 
 private:
     void readMessages();
+    std::pair<std::vector<double>, std::vector<Eigen::VectorXd>> propagator(const std::function<Eigen::VectorXd(double, Eigen::VectorXd)>& EOM, std::array<double, 2> interval, const Eigen::VectorXd& X0, double dt);
+    Eigen::VectorXd RK4(const std::function<Eigen::VectorXd(double, Eigen::VectorXd)>& ODEfunction, const Eigen::VectorXd& X0, double t0, double dt);
 
```

Comment:
The refactors in this commit could be squashed into the previous commit containg the updateState method